metadata.json

```diff
@@ -78,7 +79,8 @@
         "9",
         "10",
         "11",
-        "12"
+        "12",
+        "15"
```

Comment:
The README will also need to be updated with this info under Limitations at the bottom.